components/reddit/sources/new-link-on-a-subreddit/new-link-on-a-subreddit.js

```diff
@@ -0,0 +1,84 @@
+const reddit = require("../../reddit.app.js");
+module.exports = {
+  key: "new-link-on-a-subreddit",
+  name: "New Link on a subreddit",
```

Comment:
Sorry I missed this earlier, but our source names typically refer to the type of event they're emitting in the plural form, so I'd change this to "New Links in Subreddit" (and change the `key` accordingly).

Reply:
ahh that-s okay. know what? in the related github issue, the actions were in singular that's why I keep not using plural.

```
New Link in a Subreddit
New Comment in a Subreddit
```

All right, I made the change.